docs/orchestrating-elastic-stack-applications/elasticsearch/jvm-heap-dumps.asciidoc

```diff
@@ -20,15 +20,15 @@ To take a heap dump before the JVM process runs out of memory you can execute th
 [source,sh,subs="attributes,+macros"]
 ----
 kubectl exec $POD_NAME -- su elasticsearch -g root -c \
-  '/usr/share/elasticsearch/jdk/bin/jmap -dump:format=b,file=data/heap.hprof $(pgrep java)'
+  '/usr/share/elasticsearch/jdk/bin/jmap -dump:format=b,file=data/heap.hprof $(pgrep -n java)'
```

Comment:
```suggestion
  '/usr/share/elasticsearch/jdk/bin/jmap -dump:format=b,file=data/heap.hprof $(pgrep -f org.elasticsearch.bootstrap.Elasticsearch)'
```

Reply:
Assuming we won't change the  java class name in the future, lgtm!